partner_communication_switzerland/models/res_partner.py

```diff
@@ -55,6 +53,19 @@ def _compute_salutation(self):
         # Family shouldn't be used with informal salutation
         family = self.env.ref("partner_compassion.res_partner_title_family")
         for partner in self.filtered(lambda p: p.title == family):
+            # The family salutation have a problem here. As it is defined in the
+            # previous loop, it is considered as company
+            title = partner.title
+            title_salutation = (
+                partner.env["ir.advanced.translation"]
+                    .get("salutation", female=title.gender == "F",
+                         plural=title.plural)
+                    .title()
+            )
+            title_name = title.name
+            partner.salutation = (
+                    title_salutation + " " + title_name + " " + partner.lastname
+            )
```

Comment:
This does not change much, but the assignation to a variable with a name that have the same length and is used only once can be removed.
```suggestion
            partner.salutation = (
                    title_salutation + " " + title.name + " " + partner.lastname
            )
```